src/routes/index.ts

```diff
@@ -0,0 +1,61 @@
+import { lazy } from "react";
+
+const routes = [
+  { path: "/login", component: lazy(() => import("pages/Login")), exact: true },
+  {
+    path: "/login/privacyPolicy",
+    component: lazy(() => import("pages/Login")),
+    exact: true,
+  },
+  {
+    path: "/login/fail",
+    component: lazy(() => import("pages/Login/LoginFail")),
+    exact: true,
+  },
+  {
+    path: "/logout",
+    component: lazy(() => import("pages/Login/Logout")),
+    exact: true,
+  },
+  {
+    path: ["/", "/home"],
+    component: lazy(() => import("pages/Home")),
+    exact: true,
+  },
+  {
+    path: "/search",
+    component: lazy(() => import("pages/Search")),
+    exact: true,
+  },
+  {
+    path: "/addroom",
+    component: lazy(() => import("pages/Addroom")),
+    exact: true,
+  },
+  {
+    path: "/myroom",
+    component: lazy(() => import("pages/Myroom")),
+    exact: true,
+  },
+  {
+    path: "/myroom/:roomId",
+    component: lazy(() => import("pages/Myroom")),
+    exact: true,
+  },
```

Comment:
`lazy loading` 너무 좋은 것 같아요 !!!! 👍 👍 👍 

최적화 기술들 도입 시도해 주셔서 감사합니다 :)

```suggestion
  {
    path: ["/myroom", "/myroom/:roomId"],
    component: lazy(() => import("pages/Myroom")),
    exact: true,
  },
```

두 개 합칠 수 있을 것 같아요